README.md

```diff
@@ -158,6 +95,10 @@ Flags:
       --patch-manifests stringArray     An optional set of <kind|*>/<name|*>/path=key operations for the output manifests
 ```
 
+## Installation
+
+To install `score-k8s`, follow the instructions as described in our [installation guide](https://docs.score.dev/docs/score-implementation/score-k8s/#installation).
```

Comment:
@sujaya-sys these repos' should be self-contained to allow for forking the project correctly. I suggest we don't deduplicate this content.

Reply:
espescially the install from source

Reply:
good point!